content/get-started/climate-change-and-climate-data-in-california.md

```diff
@@ -44,18 +44,22 @@ The 2020 Adaptation Planning Guide (APG) provides guidance to local governments
+>Cities and Counties in California are required to address climate adaptation and vulnerability in their safety elements. For more information on the requirements see the State's 2020 Adaptation Planning Guide, the General Plan Guidelines Chapter 4, or Government Code section 65302(g). (Senate Bill 379, Jackson, 2015).
  
 The way that you use climate data will depend on your particular needs, risk tolerance, and planning context. To support adaptation planning, the State of California has produced guidance materials, including:
  
-- [2020 Adaptation Planning Guide](https://resilientca.org/apg/)
-- [State of California Sea Level Rise Guidance, 2018 Update](https://www.opc.ca.gov/webmaster/ftp/pdf/agenda_items/20180314/Item3_Exhibit-A_OPC_SLR_Guidance-rd3.pdf)
-- [Planning and Investing for a Resilient California: A Guidebook for State Agencies](http://opr.ca.gov/docs/20180313-Building_a_Resilient_CA.pdf)
-- [General Plan Guidelines and Technical Advisories](https://opr.ca.gov/planning/general-plan/) (See Chapter 4, Safety Element, and Chapter 8 for specific guidance on addressing climate change).
-- [Defining Vulnerable Communities in the Context of Climate Change](http://www.opr.ca.gov/docs/20200720-Vulnerable_Communities.pdf)
-- Various state agencies produce regional, topical, and impact-specific guidance documents and technical resources that are compiled and highlighted on the State Adaptation Clearinghouse as Featured Resources, organized under [Topic Areas](https://resilientca.org/topics/).
+- <a href="https://resilientca.org/apg/" target="_blank">2020 Adaptation Planning Guide</a>
+- <a href="https://www.opc.ca.gov/webmaster/ftp/pdf/agenda_items/20180314/Item3_Exhibit-A_OPC_SLR_Guidance-rd3.pdf" target="_blank">State of California Sea Level Rise Guidance, 2018 Update</a>
+- <a href="http://opr.ca.gov/docs/20180313-Building_a_Resilient_CA.pdf" target="_blank">Planning and Investing for a Resilient California: A Guidebook for State Agencies</a>
+- <a href="https://opr.ca.gov/planning/general-plan/" target="_blank">General Plan Guidelines and Technical Advisories</a> (See Chapter 4, Safety Element, and Chapter 8 for specific guidance on addressing climate change).
+- <a href="http://www.opr.ca.gov/docs/20200720-Vulnerable_Communities.pdf" target="_blank">Defining Vulnerable Communities in the Context of Climate Change</a>
+- Various state agencies produce regional, topical, and impact-specific guidance documents and technical resources that are compiled and highlighted on the State Adaptation Clearinghouse as Featured Resources, organized under <a href="https://resilientca.org/topics/" target="_blank">Topic Areas</a>.
+
+[//]: # (Using Comment Trick to End List)
```

Comment:
Neat trick @elehmer. I think the alignment issues may have been caused by a space on the line between the last list item and the next paragraph. Either way this seems to fix it.